README.md

```diff
@@ -1 +1,16 @@
-linkedin-clone-backend
+# Linkedin Clone Backend
```

Comment:
How do we want to capitalize the letters in the name?
In the frontend I was doing "LinkedIn Clone", but we just need to agree on something.
You choose

Reply:
I was going off of the logo but they seem to capitalize LinkedIn instead of Linkedin... so let’s go with that